msdt-2/untitled5.py

```diff
@@ -0,0 +1,203 @@
+
+from scipy import signal
+from scipy.io import wavfile
+import sklearn.neural_network as nn
+import sklearn.model_selection as ms
+import sklearn.preprocessing as pp
+import sklearn.metrics as mt
+import matplotlib.pyplot as plt
+import os
+import numpy as np
+
+def write_to_csv(file_names,output_file,data_array):
+    output = open(output_file, 'w')
+    output.write(",")
+    for i in file:
+        output.write(i+',')
+    for i in range(67):
+        output.write('\n'+file_names[i])
+    for j in range(67):
+        output.write(','+str(data_array[i,j]))
+    output.close()
+
+def prepare_and_print(data_array,labels_array,metric_type):
+    data_array  = (data_array-np.min(data_array))/(np.max(data_array)-np.min(data_array))#normalize data

```

Comment:
Much better if comments of statement were be above of code of the statement.

And it's good idea to formate some math calculations like this:
```suggestion
    # normalize data
    data_array  = (
        data_array
        - np.min(data_array) / (np.max(data_array) 
        - np.min(data_array)
    )
```